shell/client/shell.html

```diff
@@ -41,6 +41,9 @@
   </div>
   {{else}}
   <div id="topbar-right">
+    {{#if isDemoUser}}
+    <a id="alert" href="https://sandstorm.io/install/">Run your own »</a>
```

Comment:
I assume the target of this link will point to some new page with helpful instructions?